tests/e2e/specs/profile/applications-passwords.test.js

```diff
@@ -93,7 +74,11 @@ describe('Manage applications passwords', () => {
 
     it('should correctly revoke all the application passwords', async() => {
         await createApplicationPassword(TEST_APPLICATION_NAME);
-        await revokeAllApplicationPasswords();
+
+        await page.waitForSelector('#revoke-all-application-passwords');
+        await page.click('#revoke-all-application-passwords');
```

Comment:
Nit: For readability, we can do something like this instead:

```suggestion
        const revokeAllApplicationPasswordsButton = await page.waitForSelector('#revoke-all-application-passwords');
        await revokeAllApplicationPasswordsButton.click();
```